Makefile.am

```diff
@@ -50,3 +50,10 @@ dist_noinst_DATA = \
 	AUTHORS COPYING README \
 	example.c \
 	testinput.c
+
+if CAN_COMPILE_EXAMPLE
+EXTRA_PROGRAMS = example
+example_SOURCES = example.c
+example_LDADD = libblkmaker-$(LIBBLKMAKER_API_VERSION).la libblkmaker_jansson-$(LIBBLKMAKER_API_VERSION).la
+example_LDFLAGS = -lgcrypt
```

Comment:
This needs to become:

```
example_CFLAGS = $(LIBGCRYPT_CFLAGS)
example_LDADD = $(LIBGCRYPT_LIBS)
```

(note the LDADD should be combined with the current one)